### DISCUSSION TITLE
ReferenceError: __VUE_PROD_DEVTOOLS__ is not defined

### DISCUSSION THREAD

**Discussion by akshay-sood** [quasarframework/quasar#9662] [CLI - SSR mode] [answered]
Hi guys,

I am at a complete loss here, and need urgent help. Since the Quasar v2 RC 1, when I try to to run my compiled SSR app on server in production mode , I get this error: UnhandledPromiseRejectionWarning: ReferenceError: __VUE_PROD_DEVTOOLS__ is not defined, and page does not load.

However, if I change the node env variable to Development, the same (already complied app in dist/ssr) works fine. 

I also just tried doing this with a completely fresh installation of Quasar. Without any of my code.

On the same server, Quasar v2  beta 15 app is running fine, also, other strapi apps, and nuxt apps.

Node JS version: 14.17

Can someone please guide me to the right direction?

Error screenshot:
![image](https://user-images.githubusercontent.com/50393320/121868765-ff3f8d00-cd1e-11eb-90a7-28b904442cf5.png)

**Comment by ontwikkelfabriek** [comment]
@akshay-sood 

> Since the Quasar v2 RC 1......

What happens if you downgrade the same app to Quasar RC1 to beta 15?

> UnhandledPromiseRejectionWarning: ReferenceError: VUE_PROD_DEVTOOLS

 Do you do something explicitly in you code/config with `VUE_PROD_DEVTOOLS` related stuff?

**Reply by akshay-sood** [reply]
@ontwikkelfabriek 

> What happens if you downgrade the same app to Quasar RC1 to beta 15?

Did not try this. Will do.

> Do you do something explicitly in you code/config with VUE_PROD_DEVTOOLS related stuff?

Nope. Not at all. 

I also tried it with a blank new quasar app (quasar create app_name --branch next), cli updated to latest version as of yesterday

**Comment by rstoenescu** [comment]
What is the version for "quasar" and "@quasar/app" packages?

**Reply by akshay-sood** [reply]
@rstoenescu 

From my Package.json:

"quasar": "^2.0.0-rc.2",
"@quasar/app": "^3.0.0-rc.2",


I also tried it with a blank new quasar app (quasar create app_name --branch next), cli updated to latest version as of yesterday

**Reply by rstoenescu** [reply]
Cannot reproduce... Can you offer a step by step instruction of what you did, including your deployment setup pls?

**Comment by rstoenescu** [comment, marked as answer]
Seems to have been a problem in Vuex 4.0.1 which has been fixed in 4.0.2 - https://github.com/vuejs/vuex/issues/1991
Please upgrade to vuex 4.0.2.

**Reply by akshay-sood** [reply]
Awesome news @rstoenescu !

Also, my apologies for being a ghost for the last 3 days, I basically had an unplanned escape trip to Manali - A beautiful mountain town in the valley of mighty Himalayas - which I would be happy to take you guys if you ever visit India!

I just came back today and the first email I saw was this update from you! Tried it just now and voila, everything works charmingly!

Can't thank you enough for this awesome news :)

💯 🥇
